csharp/src/Ice/Reference.cs

```diff
@@ -179,6 +184,17 @@ public static Reference Parse(string s, Communicator communicator, string? prope
                     throw new InvalidConfigurationException($"cannot parse property `{property}'", ex);
                 }
 
+                if (invocationTimeout == null)
+                {
+                    property = $"{propertyPrefix}.InvocationTimeout";
```

Comment:
I could not find the corresponding update to the `ToProperty` method.

For `ToProperty`, I think we should generate an InvocationTimeout property only for ice1. For ice2+, the invocation-timeout option is included in the URI.

Reply:
Updated ToProperty and added a test